integrationTests/disable/src/test/java/com/grafana/opentelemetry/DisableIntegrationTest.java

```diff
@@ -0,0 +1,26 @@
+package com.grafana.opentelemetry;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.actuate.observability.AutoConfigureObservability;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+
+@SuppressWarnings("SpringBootApplicationProperties")
+@SpringBootTest(
+        classes = {HelloController.class, DemoApplication.class, OpenTelemetryConfig.class}
+
+)
+@AutoConfigureObservability
+@TestPropertySource(properties = {
+        "grafana.otlp.enabled = false",
+})
+public class DisableIntegrationTest {
```

Comment:
This is name is a little confusing since we have integration tests and we are not disabling them. 😄   Maybe `DisableOpenTelemetryTest` or `EnabledPropertySetToFalseTest`

Reply:
names are hard - good catch!